solr/core/src/java/org/apache/solr/servlet/ResponseUtils.java

```diff
@@ -28,6 +28,11 @@
 public class ResponseUtils {
   private ResponseUtils() {}
 
+  // System property to use if the Solr core does not exist or solr.hideStackTrace is not
+  // configured. (i.e.: a lot of unit test).
+  private static final boolean SYSTEM_HIDE_STACK_TRACES =
```

Comment:
this can be `Boolean.getBoolean("solr.hideStackTrace")`

Reply:
done in latest commit